plugins/tracking/hooks.js

```diff
@@ -59,6 +67,37 @@ module.exports = {
     return promiseArray;
   },
 
+  pluginsConfigured() {
+    // Ensure analytics is actually ready, because in #pluginsLoadedCheck we
+    // may skip the check so that we don't completely block the applicaiton
+    const metadata =
+      Util.findNestedPropertyInObject(this, "configuration.metadata") || {};
+
+    global.analytics.ready(() => {
+      const setContext = () => {
+        if (!global.Raven) {
+          return;
+        }
+
+        global.Raven.setTagsContext({
+          ...metadata,
+          environment: process.env.NODE_ENV,
+          dcosVersion: MetadataStore.version,
```

Comment:
@nLight on line `90` you are checking for `!MetadataStore.version` but not here, so I wonder if you are ok in having a falsey `dcosVersion`?